docs/content/latest/yugabyte-cloud/develop/hasuracloud.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Hasura Cloud
+headerTitle: Connect Hasura Cloud to Yugabyte Cloud
+linkTitle: Hasura Cloud
+description: Connect Hasura Cloud to Yugabyte Cloud.
+menu:
+  latest:
+    identifier: hasura-cloud
+    parent: yugabyte-cloud
+    weight: 700
+isTocNested: true
+showAsideToc: true
+---
+
+Use the [Hasura GraphQL Engine](https://hasura.io) with Yugabyte Cloud to power your GraphQL applications with a distributed SQL database.
+
+The following example demonstrates how to deploy an application on both Hasura Cloud and Yugabyte Cloud using the Hasura sample Realtime Poll application. This application was built using React and is powered by Hasura GraphQL Engine backed by a Yugabyte Cloud YugabyteDB cluster. It has an interface for users to cast a vote on a poll and the results are updated in the on-screen bar chart in real time. 
+
+## Prerequisites
+
+The example has the following prerequisites.
```

Comment:
Since this is set up as sub-sections, you should probably add a mini-toc here.

(This whole pre-reqs section is bigger than it needs to be, but let's just work with the structure we have)

```suggestion
The example has the following prerequisites:

* [Yugabyte Cloud account with a free tier cluster](#yugabyte-cloud-account-and-free-tier-cluster)
* [Hasura Cloud account](#hasura-cloud-account)
* [Hasura CLI](#hasura-cli)
* [Realtime Poll application](#realtime-poll-application)
```